src/chrome/content/rules/nexusmods.com.xml

```diff
@@ -1,13 +1,52 @@
-<!--
-	Mixed content:
-	- wiki
--->
-
 <ruleset name="NexusMods.com">
 	<target host="nexusmods.com" />
 	<target host="www.nexusmods.com" />
+	<target host="chivalrymw.nexusmods.com" />
+	<target host="darksouls.nexusmods.com" />
+	<target host="dev.nexusmods.com" />
+	<target host="devdocs.nexusmods.com" />
+	<target host="dragonage.nexusmods.com" />
+	<target host="dragonage2.nexusmods.com" />
+	<target host="dungeondefenders.nexusmods.com" />
+	<target host="fallenenchantress.nexusmods.com" />
+	<target host="fallout3.nexusmods.com" />
+	<target host="farcry.nexusmods.com" />
 	<target host="filedelivery.nexusmods.com" />
+	<target host="forum.nexusmods.com" />
 	<target host="forums.nexusmods.com" />
+	<target host="global.nexusmods.com" />
+	<target host="grimrock.nexusmods.com" />
+	<target host="mbwarband.nexusmods.com" />
+	<target host="mbwithfireandsword.nexusmods.com" />
+	<target host="morrowind.nexusmods.com" />
+	<target host="mountandblade.nexusmods.com" />
+	<target host="neverwinter.nexusmods.com" />
+	<target host="neverwinter2.nexusmods.com" />
+	<target host="newvegas.nexusmods.com" />
+	<target host="oblivion.nexusmods.com" />
+	<target host="pw.nexusmods.com" />
+	<target host="reskin.nexusmods.com" />
+	<target host="shadowrunreturns.nexusmods.com" />
+	<target host="skyrim.nexusmods.com" />
+	<target host="static-1.nexusmods.com" />
+	<target host="static-2.nexusmods.com" />
+	<target host="static-3.nexusmods.com" />
+	<target host="static-4.nexusmods.com" />
+	<target host="static-5.nexusmods.com" />
+	<target host="static-6.nexusmods.com" />
+	<target host="static-7.nexusmods.com" />
+	<target host="static-8.nexusmods.com" />
+	<target host="staticdelivery.nexusmods.com" />
+	<target host="tes.nexusmods.com" />
+	<target host="wfto.nexusmods.com" />
+	<target host="wiki.nexusmods.com" />
```

Comment:
Still major MCB issues

Reply:
Can you tell me which targets have MCB issues?

Reply:
Only this one `wiki.nexusmods.com` (see removed comment on top)